static/css/plugins/profilemanager.css

```
@@ -0,0 +1,1022 @@
+    font-weight: 500;
+    letter-spacing: 0px;
+    padding: 1px 6px 1px 6px;
+    text-transform: none;
+}
+
+.welcome > .form.form-manager > .welcome-catalog > .manager-layout > .manager-form-pane > .manager-edit-banner > .manager-edit-banner-exit {
+    background-color: transparent;
+    border: none;
+    color: #735200;
+    cursor: pointer;
+    font-family: Lato, -apple-system, "BlinkMacSystemFont", "Segoe UI", "Roboto", "Open Sans", "Helvetica", "Arial", sans-serif;
```

Stylelint (17.11.0)

[error] Expected no quotes around "BlinkMacSystemFont" (font-family-name-quotes)

(font-family-name-quotes)